tests/test_cache_utils.py

```diff
@@ -333,18 +333,18 @@ def test_static_cache_greedy_sampling_pad_left(self, attn_implementation):
     @parameterized.expand(["eager", "sdpa", "flash_attention_2"])
     def test_static_cache_greedy_sampling_pad_right(self, attn_implementation):
         EXPECTED_GENERATION = [
-            "The best color is\n\n\n\n\n\n\n\n\n\n",
-            "We should not undermind the issues at hand, but address them head on.\nI think",
+            "The best color isЋ the one that complements the skin tone of",
```

Comment:
```diff
-isЋ t
+is t
```
seems strange 😅 but alright

Reply:
hehe this weird one is a copy/paste 

(it has right-padding, so we should expect weird things at generation time)